Kickstarter-iOS/Features/OptimizelyFeatureFlagTools/Controller/OptimizelyFeatureFlagToolsViewControllerTests.swift

```diff
@@ -22,6 +22,7 @@ final class OptimizelyFeatureFlagToolsViewControllerTests: TestCase {
     let mockOptimizelyClient = MockOptimizelyClient()
       |> \.features .~ [
         OptimizelyFeature.commentFlaggingEnabled.rawValue: false,
+        OptimizelyFeature.consentManagementDialogEnabled.rawValue: false,
```

Comment:
Remove this test, because it's failing the CI build.
Just add a note in this [pr](https://github.com/kickstarter/ios-oss/pull/1757) for now and we'll get to correcting it when we wrap that pr.

Reply:
done!